app/models/organization.rb

```diff
@@ -15,6 +15,8 @@ class Organization < ActiveRecord::Base
   has_one :background, -> { where(type: "organization_background") }, class_name: "Medium", as: :linked
   has_many :organization_roles, -> { where.not(roles: []) }, class_name: "AccessControlList", as: :resource
   has_many :pages, class_name: "OrganizationPage", dependent: :destroy
+  has_many :attached_images, -> { where(type: "organization_attached_image") }, class_name: "Medium",
```

Comment:
maybe just use "org_attached_image" or something that conveys the table link correctly to keep the index size down

Reply:
✔️